common/src/main/java/org/apache/rocketmq/common/TopicConfig.java

```diff
@@ -29,6 +29,7 @@
     private TopicFilterType topicFilterType = TopicFilterType.SINGLE_TAG;
     private int topicSysFlag = 0;
     private boolean order = false;
+    private Integer expirationTime;
 
```

Comment:
“expirationTime” get me confused, expirationTimeInMs can be better

Reply:
nice  suggestion